app/views/browse/_replica-set-details.html

```diff
@@ -198,7 +198,7 @@ <h3>Autoscaling</h3>
          ng-href="project/{{projectName}}/edit/autoscaler?kind=ReplicationController&name={{replicaSet.metadata.name}}"
          role="button">Add Autoscaler</a>
       <a ng-if="replicaSet.kind === 'ReplicationController' && deploymentConfigName"
-         ng-href="project/{{projectName}}/edit/autoscaler?kind=DeploymentConfig&name={{deploymentConfigName}}"
+         ng-href="project/{{projectName}}/edit/autoscaler?kind=DeploymentConfig&name={{deploymentConfigName}}&group=apps.openshift.io"
```

Comment:
Wondering if we should have things like `&group=apps.openshift.io` in our templates, or if that makes it easy to miss things later.  Could we potentially keep this in a variable in the controller near the `preferredVersion` stuff?

Just trying to think if we can avoid potential mismatches in the future, though I'm sure group changes will be fairly minimal (this is more of a question than a request for change).